setup.cfg

```diff
@@ -30,13 +30,14 @@ package_dir =
 # DON'T CHANGE THE FOLLOWING LINE! IT WILL BE UPDATED BY PYSCAFFOLD!
 setup_requires = pyscaffold>=3.2a0,<3.3a0
 # Add here dependencies of your project (semicolon/line-separated), e.g.
-install_requires = 
+install_requires =
+    aleph-sdk-python@git+https://github.com/aleph-im/aleph-sdk-python.git@main
```

Comment:
shouldn't it be the current version on PyPI instead?

Reply:
The version on PyPI is outdated, use the `master` from git for now.

I will upgrade it this week.